### ISSUE TITLE
Memory Display Size and Dim Zeros (Enhancement)

### ISSUE THREAD

**Issue by Yazwh0** [indigodarkwolf/box16#81]
While using the memory display it occurs there are a couple of enhancements that could help.

- Make the memory display resize when the windows expands, so we can see more memory data.
- Have zeros dimmed out, so make actual data easier to see. (8s and 0s are quite similar.)

![image](https://user-images.githubusercontent.com/90315107/215261841-34976c0d-03ab-4817-9e4c-17e5d540c720.png)

With zeros dimmed:
![image](https://user-images.githubusercontent.com/90315107/215261857-57e25fe9-d020-4880-8a42-8b087d19f8fe.png)

**Comment by indigodarkwolf** [comment]
I agree, that could make it a little easier to read. I can probably dim the zeroes in a quick change. Resizing the display might not take too long either now that I've discovered `ImGui::GetContentRegionAvail()` to size things within the remainder of a window's space.
